commcare-cloud/README.md

```diff
@@ -0,0 +1,84 @@
+# Set up commcarehq-ansible
+
+Clone the commcarehq-ansible repo (suggested location alongside commcare-hq):
+
+```
+git clone https://github.com/dimagi/commcarehq-ansible.git
+cd commcarehq-ansible
+```
+
+Now make a virtualenv for ansible:
+
+```
+mkvirtualenv ansible
+```
+
+If you want `workon ansible` to always bring you to this directory, then you can also run
+
+```
+pwd > ~/.virtualenvs/ansible/.project
+```
+
+at this time.
+
+**Note**: If you already have commcarehq-ansible cloned, then just enter that directory
+and update it with
+
+```
+git pull
+```
+
+# Install commcare-cloud
+
+```
+workon ansible
+```
+
+```
+pip install pip --upgrade
+```
+
+```
+pip install -e commcare-cloud
+pip install -r ansible/requirements.txt  # to be able to run ansible commands
+ansible-galaxy install -r ansible/requirements.yml  # to be able to run ansible commands
+pip install -r fab/requirements.txt  # to be able to run fab commands
+```
+
+Finally, finish your install with
+
+```
+./control/check_install.sh
+```
+
+# Optional: Hook up the bells and whistles
+
+If you want to have access to the `commcare-cloud` command outside your virtualenv,
+you can link the executable (within your vitualenv) to somewhere on your PATH.
+While it's technically redundant, I like to accomplish this by adding the link command
+to my bash profile:
+
+```
+ln -sf ~/.virtualenvs/ansible/bin/commcare-cloud /usr/local/bin
+```
```

Comment:
Personally I like the idea of adding that `bin` dir to your path rather than symlinking stuff:
```export PATH="~/virtualenvs/ansible/bin:$PATH"```
no strong feelings though

Reply:
Yeah this occurred to me early on, but I ruled it out pretty immediately because of the other things in that directory, which include e.g. `python`. I think making this always on the path would lead to confusing behavior and overriding other programs like this that people aren't expecting.